lib/rules/jsx-no-literals.js

```diff
@@ -26,6 +26,9 @@ module.exports = {
       properties: {
         noStrings: {
           type: 'boolean'
+        },
+        allowedStrings: {
+          type: 'array'
```

Comment:
this array should probably be required to be unique, and its contents of type string

Reply:
updated